src/main/java/io/snyk/snyk_maven_plugin/goal/SnykMojoExecutor.java

```diff
@@ -32,14 +36,28 @@ public void execute() throws MojoFailureException, MojoExecutionException {
 
     private int executeCommand() throws MojoExecutionException {
         try {
+            Log log = mojo.getLog();
+
+            String snykExecutablePath = mojo.getExecutable().orElseGet(this::downloadExecutable).getAbsolutePath();
+            log.info("snyk executable path: " + snykExecutablePath);
+
+            ProcessBuilder versionCommandLine = CommandLine.asProcessBuilder(
+                    snykExecutablePath,
+                    Command.VERSION,
+                    Optional.empty(),
+                    emptyList(),
+                    mojo.supportsColor()
+            );
+            log.info("snyk version:");
```

Comment:
Might be worht changing this to indicate this is an output of a `snyk (--)version` command:

```suggestion
            log.info("`snyk --version` result:");
```

Or just making sure it's clear this is a CLI version:

```suggestion
            log.info("Snyk CLI version:");
```